src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg + secondLeg) / 2 * height;
```

Comment:
The calculation of the area is correct, but be cautious with integer division. It's better to cast one of the operands to double to avoid losing the fractional part. For example: return (firstLeg + secondLeg) / 2.0 * height;